src/Config.php

```diff
@@ -187,21 +187,15 @@ private function setSender($c)
             $default = "Rollbar\Senders\FluentSender";
 
             if (isset($c['fluent_host'])) {
-                $c['senderOptions'] = array(
-                    'fluentHost' => $c['fluent_host']
-                );
+                $c['senderOptions']['fluentHost'] = $c['fluent_host'];
             }
 
             if (isset($c['fluent_port'])) {
-                $c['senderOptions'] = array(
-                    'fluentPort' => $c['fluent_port']
-                );
+                $c['senderOptions']['fluentPort'] = $c['fluent_port'];
             }
 
             if (isset($c['fluent_tag'])) {
-                $c['senderOptions'] = array(
-                    'fluentTag' => $c['fluent_tag']
-                );
+                $c['senderOptions']['fluentTag'] = $c['fluent_tag'];
```

Comment:
Oh I see. I misread this code. I though it was just usage of the [] syntax for array definition for some reason. Yeah, then this is good to go. I'll submit a github issue for this pull request.